app/decorators/decidim/newsletters_helper_decorator.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+Decidim::Admin::NewslettersHelper.class_eval do
+  alias_method :original_spaces_user_can_admin, :spaces_user_can_admin
+
+  def spaces_user_can_admin
+    author= current_user
+
+    if author&.department_admin?
+      return spaces_department_admin_can_admin
+    end
+
+    original_spaces_user_can_admin
```

Comment:
I like better the following
```suggestion
    if author&.department_admin?
      spaces_department_admin_can_admin
    else
      original_spaces_user_can_admin
    end
```